src/GLideNUI/GLideNUI.cpp

```diff
@@ -96,6 +81,28 @@ int runAboutThread(const wchar_t * _strFileName) {
 	return 0;
 }
 
+EXPORT void CALL ConfigInit(void * hinst)
+{
+	initMyResource();
+
+	s_pAppInstance = QCoreApplication::instance();
+
+	if (s_pAppInstance == nullptr) {
+		int argc = 0;
+		char * argv = 0;
+		s_pQApp.reset(new QApplication(argc, &argv));
+		s_pAppInstance = s_pQApp.get();
+	}
+}
+
+EXPORT void CALL ConfigCleanup(void)
+{
+	cleanMyResource();
+	if (s_pQApp) {
```

Comment:
Redundant check. You can safely release an empty smart pointer.

Reply:
Removed the check, thank you!